Here's the [branch](https://github.com/hyleung/ratpack-zipkin-example/tree/upgrade-to-v2) in my test project where I've been trying to upgrade to v2. When I *don't* include the v1 reporter dependency, I get a compilation error:

```
./gradlew clean check
:clean
:compileJava
/Users/hyleung/Projects/ratpack-zipkin-example/src/main/java/com/example/server/App.java:29: error: cannot access Reporter
                  .spanReporter(spanReporter());
                  ^
  class file for zipkin.reporter.Reporter not found
1 error
:compileJava FAILED
```

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,4 +1,4 @@
 version=2.1.1-SNAPSHOT
 
-braveVersion=4.9.1
+braveVersion=4.9.2
 ratpackVersion=1.4.6

--- a/src/main/java/ratpack/zipkin/ServerTracingModule.java
+++ b/src/main/java/ratpack/zipkin/ServerTracingModule.java
@@ -147,16 +147,21 @@ public class ServerTracingModule extends ConfigurableModule<ServerTracingModule.
      *
      * If not set, defaults to {@link Reporter#NOOP}.
      *
-     * @param reporter the reporter
+     * @param reporter a V2 reporter
      *
      * @return the config
      */
-    public Config spanReporter(final Reporter<Span> reporter) {
+    public Config spanReporterV2(final Reporter<Span> reporter) {
       this.spanReporter = reporter;
       return this;
     }
 
-    /** @deprecated please use {@link #spanReporter(Reporter)} */
+    /** @deprecated please use {@link #spanReporterV2(Reporter)}
+     *
+     * @param reporter a V1 reporter
+     *
+     * @return the config
+     * */
     // Until this is removed, we need a mandatory dep on io.zipkin.reporter:zipkin-reporter
     // due to overloading requiring access to all overloaded types
     @Deprecated
@@ -267,8 +272,8 @@ public class ServerTracingModule extends ConfigurableModule<ServerTracingModule.
      *
      * Defaults to {@link B3Propagation.Factory}.
      *
-     * @param propagationFactory
-     * @return
+     * @param propagationFactory the Zipkin propagation factory
+     * @return the config
      */
     public Config propagationFactory(final Propagation.Factory propagationFactory) {
       this.propagationFactory = propagationFactory;

--- a/src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy
+++ b/src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy
@@ -1,7 +1,6 @@
 package ratpack.zipkin
 
 import brave.SpanCustomizer
-import brave.Tracing
 import brave.http.HttpSampler
 import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.MockWebServer
@@ -62,7 +61,7 @@ class ServerTracingModuleSpec extends Specification {
 								.sampler(Sampler.create(1f))
 								.clientSampler(HttpSampler.TRACE_ID)
 								.serverSampler(HttpSampler.TRACE_ID)
-								.spanReporter(Reporter.NOOP)
+								.spanReporterV2(Reporter.NOOP)
 					})
 				}).handlers {
 					chain ->
@@ -83,7 +82,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(zipkin.reporter.Reporter.NOOP)
+								.spanReporterV2(zipkin.reporter.Reporter.NOOP)
 					})
 				}).handlers {
 					chain ->
@@ -141,7 +140,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -168,7 +167,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -206,7 +205,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -247,7 +246,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -276,7 +275,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -312,7 +311,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -351,7 +350,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.create(1f))
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -386,7 +385,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.NEVER_SAMPLE)
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -410,7 +409,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.NEVER_SAMPLE)
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -446,7 +445,7 @@ class ServerTracingModuleSpec extends Specification {
 						config
 								.serviceName("embedded")
 								.sampler(Sampler.ALWAYS_SAMPLE)
-								.spanReporter(reporter)
+								.spanReporterV2(reporter)
 					})
 				}).handlers {
 					chain ->
@@ -484,7 +483,7 @@ class ServerTracingModuleSpec extends Specification {
 					config
 							.serviceName("embedded")
 							.sampler(Sampler.create(1f))
-							.spanReporter(reporter)
+							.spanReporterV2(reporter)
 				})
 			}).handlers {
 				chain ->
@@ -517,7 +516,7 @@ class ServerTracingModuleSpec extends Specification {
                         config
                             .serviceName("embedded")
                             .sampler(Sampler.ALWAYS_SAMPLE)
-                            .spanReporter(reporter)
+                            .spanReporterV2(reporter)
                             .spanNameProvider(new SpanNameProvider() {
                             @Override
                             String spanName(

